db/sql/sql_test.go

```diff
@@ -632,6 +632,68 @@ var _ = Describe("Sql", func() {
 				Expect(resSql).To(Equal(expectedSql))
 				Expect(param).To(Equal(expectedParam))
 			})
+			It("should handle empty list in WHERE ... IN ()", func() {
```

Comment:
Since all of these tests have the same structure:
```
filter := map[string]interface{}{...}

res, err := AddFilterToQuery(testSchema, query, filter, false)

Expect(err).ToNot(HaveOccurred())
resSql, param, err := res.ToSql()
Expect(err).ToNot(HaveOccurred())

expectedQuery = expectedQuery.Where(...)
expectedSql, expectedParam, _ := expectedQuery.ToSql()
Expect(resSql).To(Equal(expectedSql))
Expect(param).To(Equal(expectedParam))
```
I would consider grouping them into `DescribeTable`

Reply:
sure, done